shared/src/iosMain/kotlin/com/vml/tutorial/plantshop/core/utils/DateFormat.kt

```diff
@@ -0,0 +1,24 @@
+package com.vml.tutorial.plantshop.core.utils
+
+import platform.Foundation.NSCalendar
+import platform.Foundation.NSDate
+import platform.Foundation.NSDateComponents
+import platform.Foundation.NSDateFormatter
+import platform.Foundation.NSLocale
+import platform.Foundation.currentLocale
+
+actual fun formatDate(dateMillis: Long, dateFormat: String): String {
+    val components = NSDateComponents().apply {
+        year = 1970
+        month = 1
+        day = 1
+        second = (dateMillis / 1000)
+    }
+    val dateFormatter = NSDateFormatter().apply {
+        this.dateFormat = dateFormat
+        locale = NSLocale.currentLocale
+    }
+    val date = NSCalendar.currentCalendar.dateFromComponents(components)
+
+    return dateFormatter.stringFromDate(date ?: NSDate())
```

Comment:
Fun fact: yesterday I created a similar method :D nevermind you were faster :P but Can I show you a simple version of this?

        return NSDateFormatter().apply {
            dateFormat = format
            locale = NSLocale.currentLocale
        }.stringFromDate(NSDate.dateWithTimeIntervalSince1970(dateMillis.div(1000.0)))

Reply:
Thank you for sharing. I wasn't previously acquainted with the dateWithTimeIntervalSince1970 function. Refactored :)